packages/babel-plugin-proposal-object-rest-spread/test/fixtures/object-rest/for-x-completion-record/output.js

```diff
@@ -0,0 +1,9 @@
+for (var _ref of []) {
+  var _ref2 = _ref;
```

Comment:
Is it possible to avoid this double temp var? (Not a big deal)

Reply:
I suppose we could add the `isCompletionRecord` check from `destructuring` to inhibit repeating the expression and thus lose the second temp var in the general case. The problem is that's actually an unsound check for what it tries to achieve (which came up on another PR just now).

Reply:
It's ok like this then